dizoo/setup.py

```diff
@@ -0,0 +1,6 @@
+from setuptools import setup
```

Comment:
remove this setup.py, you have not installed any packages and the requirements is already indicated in `DI-engine/setup.py`